ra/ra.go

```diff
@@ -847,19 +881,32 @@ func (ra *RegistrationAuthorityImpl) FinalizeOrder(ctx context.Context, req *rap
 	}
 	cert, err := ra.issueCertificate(ctx, issueReq, accountID(*order.RegistrationID), orderID(*order.Id))
 	if err != nil {
+		// Fail the order. The problem is computed using
```

Comment:
Again this seems like we should only fail the order if the error isn't a 500, although I guess since it's already been set to the processing state it cannot be retried in that case?

Reply:
Replied in [the other comment thread](https://github.com/letsencrypt/boulder/pull/3404#discussion_r166320683).